.github/workflows/release.yml

```
@@ -81,7 +81,7 @@ jobs:
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-region: ${{ env.AWS_DEFAULT_REGION }}
           role-to-assume: ${{ secrets.BUILD_ROLE_TO_ASSUME_PRODUCTION }}
-          role-duration-seconds: 3600
+          role-duration-seconds: 7200  # 2 hours
```

This duration should at least be the same as in `prerelease.yml` (if not longer, but it sounds like you have given us adequate cushion with this duration):
```suggestion
          role-duration-seconds: 14400  # 4 hours
```